versioned_docs/version-2.5/pages-for-subheaders/integrations-in-rancher.md

```diff
@@ -2,6 +2,10 @@
 title: Integrations in Rancher
 ---
 
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/getting-started/overview"/>
```

Comment:
```suggestion
  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/pages-for-subheaders/cloud-marketplace"/>
```

Since our latest version has a sidebar category labeled exactly as "Integrations in Rancher", we point as close to that as possible. It's not a perfect solution, but cloud-marketplace is the first page under the Integrations category rather than a completely different area of the site.

Also, the 2.0-2.4 version of this page is missing from the PR.